src/gt4py/next/utils.py

```diff
@@ -140,7 +140,7 @@ def impl(*args: Any | tuple[Any | tuple, ...]) -> _R | tuple[_R | tuple, ...]:
                 assert result_collection_constructor is not None
                 return result_collection_constructor(impl(*arg) for arg in zip(*args))
 
-            return fun(  # type: ignore[misc] # mypy not smart enough
+            return fun(  # type: ignore[call-arg, misc] # mypy not smart enough
```

Comment:
The cast is probably superfluous together with `call-arg`, but whatever.

Reply:
I double checked. The cast actually fixes a `arg-type` error. The `call-arg` is for "too few arguments", not motivated to investigate further.